client/src/components/screens/VariantDetails/index.jsx

```diff
@@ -136,30 +136,30 @@ const impact = (c) => {
 
   if (c.predictions) {
     const sift = c.predictions.SIFT
-      ? (<li><span className="consequenceTerm">SIFT: </span>{c.predictions.SIFT} - {c.predictions.SIFT_score}</li>) : null;
+      ? (<li><span className="consequenceTerm">SIFT: &nbsp;</span>{c.predictions.SIFT} - {c.predictions.SIFT_score}</li>) : null;
```

Comment:
@latch2112 We should probably the CSS instead, but this works too...

Reply:
I removed it because it's not part of the ticket